app/services/scrape.py

```diff
@@ -111,16 +113,15 @@ def get_user_by_id(self, twitter_id: str) -> TwitterUser:
                     avatar=user.profile_image_url,
                     banner=getattr(user, "profile_banner_url", None),
                     tweets=recent_tweets,
-                    score=ml_service.get_analysis_result(user.screen_name),
+                    score=self._ml_service.get_analysis_result(user.screen_name),
                 )
 
                 user_db.save()
 
         return user_db
 
     def get_followers(self, followers_numbs):
-        """
-        Get list of Twitter user's followers.
+        """Get list of Twitter user's followers.
```

Comment:
ugg what :v?

Reply:
I thought u not gonna care about that :V

Reply:
https://peps.python.org/pep-0257/

Reply:
Huh the doc say that both formats are the same? :v

> The summary line may be on the same line as the opening quotes or on the next line.